modules/nw-ovn-kubernetes-migration.adoc

```diff
@@ -66,7 +113,30 @@ do
 done
 ----
 
-. After the nodes in your cluster have rebooted, confirm that the migration succeeded:
+. After the nodes in your cluster have rebooted, start all of the machine configuration pools. As the MCO updates machines in each config pool, by default it updates a single machine per pool at a time, so the time that the migration requires to complete grows with the size of the cluster.
```

Comment:
To apply the new MachineConfig to a node, the MCO will trigger a reboot for each node.

Reply:
@pliurh updated to

> After the nodes in your cluster have rebooted, start all of the machine configuration pools. As the MCO updates machines in each config pool, it reboots each node. By default the MCO updates a single machine per pool at a time, so the time that the migration requires to complete grows with the size of the cluster.

Reply:
Here comes the output example of `oc describe node | egrep "hostname|machineconfig"` for one node.

```
                    kubernetes.io/hostname=master-0
                    machineconfiguration.openshift.io/currentConfig: rendered-master-c53e221d9d24e1c8bb6ee89dd3d8ad7b
                    machineconfiguration.openshift.io/desiredConfig: rendered-master-c53e221d9d24e1c8bb6ee89dd3d8ad7b
                    machineconfiguration.openshift.io/reason:
                    machineconfiguration.openshift.io/state: Done
```

When the `state` is done, the `currentConfig` shall equal to the `desiredConfig`. The name of the MachineConfig can be found by `oc get machineconfig`. When we trigger the migration, a new MachineConfig will be rendered by MCO for each MachineConfigPool. Users have to make sure that the newly rendered MachineConfig has been applied to the node successfully by comparing the name of `currentConfig` and the output of `oc get machineconfig`.

```
[root@nfvpe-21 ~]# oc get mc
...
rendered-master-c53e221d9d24e1c8bb6ee89dd3d8ad7b    6c8bd9b2915c44e03ec10e02e59fedb07e3807cb   3.1.0             6d23h
...
```